src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrdersApiToOrderDataMapper.java

```diff
@@ -1,10 +1,63 @@
 package uk.gov.companieshouse.itemhandler.mapper;
 
+import org.mapstruct.AfterMapping;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
 import uk.gov.companieshouse.api.model.order.OrdersApi;
+import uk.gov.companieshouse.api.model.order.item.BaseItemApi;
+import uk.gov.companieshouse.api.model.order.item.BaseItemOptionsApi;
+import uk.gov.companieshouse.api.model.order.item.CertificateItemOptionsApi;
+import uk.gov.companieshouse.api.model.order.item.CertifiedCopyItemOptionsApi;
+import uk.gov.companieshouse.itemhandler.model.CertificateItemOptions;
+import uk.gov.companieshouse.itemhandler.model.CertifiedCopyItemOptions;
+import uk.gov.companieshouse.itemhandler.model.Item;
+import uk.gov.companieshouse.itemhandler.model.ItemOptions;
+import uk.gov.companieshouse.itemhandler.model.ItemType;
 import uk.gov.companieshouse.itemhandler.model.OrderData;
 
 @Mapper(componentModel = "spring")
 public interface OrdersApiToOrderDataMapper {
     OrderData ordersApiToOrderData(OrdersApi ordersApi);
+
+    @Mapping(source = "id", target="id")
+    @Mapping(source = "companyName", target="companyName")
+    @Mapping(source = "companyNumber", target="companyNumber")
+    @Mapping(source = "customerReference", target="customerReference")
+    @Mapping(source = "description", target="description")
+    @Mapping(source = "descriptionIdentifier", target="descriptionIdentifier")
+    @Mapping(source = "descriptionValues", target="descriptionValues")
+    @Mapping(source = "itemCosts", target="itemCosts")
+    @Mapping(source = "etag", target="etag")
+    @Mapping(source = "kind", target="kind")
+    @Mapping(source = "postalDelivery", target="postalDelivery")
+    @Mapping(source = "quantity", target="quantity")
+    @Mapping(source = "links", target="links")
+    @Mapping(source = "postageCost", target="postageCost")
+    @Mapping(source = "totalItemCost", target="totalItemCost")
+    @Mapping(source = "links.self", target="itemUri")
+    @Mapping(target = "satisfiedAt", ignore = true)
+    @Mapping(target = "status", ignore = true)
```

Comment:
Are many of these potentially redundant mapping configs?

Reply:
Yes, they're not needed. I've removed them. Thanks